lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -502,7 +504,7 @@ extension DomHTMLImageElementExtension on DomHTMLImageElement {
 class DomHTMLScriptElement extends DomHTMLElement {}
 
 extension DomHTMLScriptElementExtension on DomHTMLScriptElement {
-  external set src(String value);
+  external set src(Object /*String|TrustedScriptURL?*/ value);
```

Comment:
Is it `TrustedScriptURL?`, i.e. nullable? If so, then should the Dart type be `Object?`?

Reply:
It *is* nullable, but we never set it to `null`. The Trusted Types API returns an empty string `''` rather than `null` when returning `null` from the function that creates the trusted URL. (AFAIK this is not documented in the API anywhere :/)